replate/src/components/Login.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { login } from '../actions';
+
+class Login extends React.Component {
+    state = {
+        credentials: {
+            username: '',
+            password: ''
+        }
+    };
+
+    handleChanges = e => {
+        this.setState ({
+            credentials: {
+                ...this.state.credentials,
+                [e.target.name]: e.target.value
+            }
+        });
+    };
+
+    login = e => {
+        e.preventDefault();
+        this.props.login(this.state.credentials).then(() => {
+            this.props.history.push(``);
```

Comment:
Redirect is also a nice way to accomplish this, though not always the most straightforward.
https://tylermcginnis.com/react-router-programmatically-navigate/